docs/contributing/submit-to-starter-library.md

```diff
@@ -42,6 +42,8 @@ Use the following template to ensure required fields are filled:
 
 Check out the [`starters.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/starters.yml) file for examples.
 
+Submit your pull request with title `chore(starters): add my-starter-name-here`. If lint complains about your PR, make sure that the file ends with exactly one newline character (no spaces), and that your text values don't contain special characters like `:`.
```

Comment:
`lint complains about your PR, make sure that the file ends with exactly one newline character (no spaces)`
```suggestion
We prefer pull requests with the title in the following format:
```chore(starters): add my-starter-name-here```
If there are linting issues with your PR, you can fix them by running `npm run format`. 
```

Small suggestion to recommend the format script instead since Prettier config could change or later versions of Prettier might have slightly different behaviour.